### ISSUE TITLE
Allow classes for window_lsm()

### ISSUE THREAD

**Issue by mhesselbarth** [r-spatialecology/landscapemetrics#211]
Allow also class level metrics for window lsm. 

Either specify focal class beforehand or return RasterStack in which each layer represents a class.

**Comment by Nowosad** [comment]
I would suggest the second option (RasterStack) or, if possible both options. 

(We should also wait for the email reply before committing to any solutions...)

**Comment by mhesselbarth** [comment]
I think you were not included in the reply, sorry (I just forwarded it to you). 

I think something similar to e.g. the `class` argument in e.g. `show_lsm()` would be cool. So basically it possible to get all classes or only selected ones. From a computational point, we need to calculate all classes anyways and then either just return one specific class or all.

**Comment by bitbacchus** [comment]
> From a computational point, we need to calculate all classes anyways and then either just return one specific class or all.

**Comment by bitbacchus** [comment]
> From a computational point, we need to calculate all classes anyways and then either just return one specific class or all.

I don't think so. I'm just hacking a class-level version for `ed` for myself and I just reclassify everything except the focal class to `NA`. Then I added an optional ` count_background_NA` option to `lsm_l_ed`:
